crates/moonutil/src/scan.rs

```diff
@@ -334,12 +334,17 @@ fn scan_one_package(
             )
         }))
     };
+    let name_path = PathComponent::from_str(&mod_desc.name)?;
```

Comment:
how about make this verify at 
https://github.com/moonbitlang/moon/blob/62b13a67ea6caac3254895343f3ca70a87c24339/crates/moonutil/src/module.rs#L532